css/menu.css

```diff
@@ -114,39 +168,325 @@ header {
     width: 100%;
     height: auto;
     background-image: url('../img/staff-bartender.jpg');
-    background-color: rgb(0, 4, 17);
-    background-size: cover;
+    background-size:cover;
     background-position: center;
     background-repeat: no-repeat;
+    text-align: center;
 }
 
+.menu .menu-container{
+    display:flex;
+    justify-content: center;
+    flex-wrap: wrap;
+    padding: 1%;
+    background-color: rgb(0, 4, 17);
+    opacity: 0.8;
+}
+
+.menu .menu-container h2{
+    width:100%;
+    display: flex;
+    justify-content: center;
+    font-size:4rem;
+}
+
+.menu .menu-container .menu-section{
+    width: 50%;
+    display:flex;
+    justify-content: space-around;
+    flex-wrap:wrap;
+    padding: 2% 10%;
+}
+
+.menu-section h3{
+    display:flex;
+    width:100%;
+    justify-content: flex-start;
+    flex-wrap:wrap;
+    font-size:2.5rem;
+    padding: 2% 0;
+}
+
+.menu-item{
+    width:100%;
+    display: flex;
+    flex-wrap:wrap;
+    padding:2% 0;
+    text-align: center;
+}
+
+.menu-item h4{
+    width:50%;
+    display:flex;
+    align-items: space-evenly;
+    font-size: 2rem;
+}
+
+.menu-item .price{
```

Comment:
I see what you did here and it worked for all but the Flourless Chocolate cake item 

An approach that would allow for all items to look good would be to wrap a div around the h4 title and h4 price and put a flex on that div and take away the width limitation on the .menu-item h4 

So we added a div with a class of .name-price around the h4 name and h4 price it would look like

`.name-price {
    display: flex;
    justify-content: space-between;
    width: 100%;
}

.menu-item h4{
    text-align: left;
    font-size: 2rem;
}
  `

Reply:
Definitely a good start / excellent implementation of what you were given with your current knowledge.